day_two_guide/topics/etcd_backup.adoc

```diff
@@ -188,9 +209,11 @@ Snapshot saved at /backup/etcd-<date>/db
 The `etcdctl snapshot save` command requires the etcd service to be running.
 ====
 +
-In this example, a `/backup/etcd-<date>/` directory is created, where `<date>`
+--
+In these commands, a `/backup/etcd-<date>/` directory is created, where `<date>`
 represents the current date, which must be an external NFS share, S3 bucket, or
 any external storage location.
-+
+
 In the case of an all-in-one cluster, the etcd data directory is located in
-`/var/lib/origin/openshift.local.etcd`
+the `/var/lib/origin/openshift.local.etcd` folder.
```

Comment:
I'd say s/folder/directory.

I can't find where the standard is written down, but you'll have to trust me that it's a thing :)

Reply:
I don't see it in the IBM Style Guide or our glossary. It's probably an issue about precision, since a folder is not necessarily a directory, even though they're often used interchangeably. (It might be a good addition to our glossary, though!)